packages/angular_bloc/pubspec.yaml

```diff
@@ -10,7 +10,8 @@ environment:
 
 dependencies:
   angular: ^5.3.0
-  bloc: ^3.0.0
+  bloc:
```

Comment:
same comment here about why this is needed?

Reply:
Reverted